backport-Guide.md

```diff
@@ -0,0 +1,97 @@
+
+## Backport Workflow
+
+The basic workflow involves creating a new local branch from the stable tree you
+are targetting, then cherry picking the commits from your master branch PR
+onto that branch. You then submit your branch to GitHub as a PR against the
+stable branch (and not the `master` branch).
+
+The following example shows a pseudo-backport of a PR from the `master` branch
+into the `stable-1.2` branch of the `runtime` repo.
+
+- Ensure your local repo is up to date:
+
+    ```bash
+    $ cd ${GOPATH}/src/github.com/kata-containers/runtime
+    $ git fetch origin
+    ```
+
+    And check the list of current stable branches:
+
+    ```bash
+    $ cd ${GOPATH}/src/github.com/kata-containers/runtime
+    $ git branch -r | grep origin/stable
+    ```
+
+- Create your branch to work on:
+
+    ```bash
+    $ git checkout -b my_1.2_pr_backport origin/stable-1.2
+    ```
+
+- Locate the commits you want to pull in:
+
+    Here we look in the `master` branch, presuming the PR has been merged.
+    If you have the PR in a local branch you can substitute `master` for the name
+    of that branch.
+
+    ```bash
+    $ git log --oneline master
+    # And search for the SHA of the commits you wish to backport.
+    ```
+
+- Pull in your commits:
+
+    If you are pulling the commits in from the `master` branch, you can add the `-x`
+    argument to `git cherry-pick` to automatically add a reference in the
+    commit to the original commits. This is strongly recommended to aid traceability.
+    If you pick the commits from your local branch do *not* use `-x`. This
+    potentially adds references to SHAs that only exist in your user branches, which
+    is not be useful for future tracability.
+
+    Also, it is required you add the `-s` signoff to the commits, if you did not
+    create the original commits.
+
+    ```bash
+    $ git cherry-pick -x <commit>...
+    ```
+
+    You can cherry pick ranges of commits, etc. Please see the `git-cherry-pick(1)`
+    man page for more information.
+
+- Resolve any conflicts, etc.:
+
+    You might encounter conflicts during your cherry pick, which need to be resolved
+    before you continue. Follow standard practices for git conflict resolution, and see
+    the guidance printed by `git cherry-pick` on processing and applying those fixes.
+
+    If you hit a conflict, any effects of `-x` to `cherry-pick` might not be
+    applied. In this case, consider hand-adding the `master` SHA references and a note
+    that you resolved a conflict to the commit message.
+
+- Test your changes:
+
+    Before you push your changes, you should test that they work and nothing has been
+    broken. No matter how small the change, running the test suite is always recommended.
+
+- Push your branch to your GitHub repo:
+
+    ```bash
+    $ git push my_remote my_1.2_pr_backport
+    ```
+
+- Submit a PR from your branch to *the stable branch*:
```

Comment:
Why the italic on "stable branch"?

Reply:
I was trying to make it clear you push it to the stable branch, not the master. I could make it **bolder**, or drop it, or leave as is. Any prefs?

Reply:
Ok I see. I am not a UX expert so I am fine leaving as it is, assuming it is coherent with the rest of the doc :)